icechunk-python/python/icechunk/_icechunk_python.pyi

```diff
@@ -275,6 +279,56 @@ class StoreConfig:
         """
         ...
 
+class VersionSelection:
```

Comment:
Is this better than an enum?

Reply:
This is how Rust enums map unfortunately (i think, I'll confirm)